source/user-manual/wazuh-indexer/certificates.rst

```diff
@@ -105,4 +105,27 @@ After running the script, the directory ``wazuh-certificates`` will be created a
         ├── indexer-key.pem
         ├── indexer.pem
         ├── server-key.pem
-        └── server.pem
\ No newline at end of file
+        └── server.pem
+
+.. _wazuh_cert_tool_docker:
+
+Wazuh certificates tool in Docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Wazuh certificates tool can also be used in a Docker container. To do so, run the following command:
+
+    .. code-block:: console
+
+        # docker run -v /path/to/config.yml:/config/certs.yml -v /path/to/wazuh-certificates/:/certificates/ -itd wazuh/wazuh-cert-tool
+
+The Docker container must have access to the ``config.yml`` file and the directory where the certificates will be stored. To achieve this, the ``-v`` option is used to mount the directories.
+
+  .. warning::
+
+      The ``/path/to/config.yml`` and ``/path/to/wazuh-certificates/`` directories must be replaced with the actual paths in the host machine. We recommend using absolute paths and the certificates directories must be named ``wazuh-certificates``.
+
+After running the command, the container will be created and the certificates will be stored in the directory specified.
```

Comment:
Please, change:
`stored in the directory specified.`
For:
`stored in the specified directory.`

Reply:
Done: 412b9ae0f